README.md

```diff
@@ -128,6 +140,17 @@ There are multiple ways to get involved and learn more about our work. That's ho
       Java HotSpot(TM) 64-Bit Server VM (build 25.341-b10, mixed mode)
    ```
 
+## Running the model in previous versions
+
+The model can be run in previous versions by checking out the respective tag. For instance, to run the model in version 0.4.0, which is the last version before the repo `pypsa-earth-sec` was merged, the following command can be used:
```

Comment:
Just as a flag, this will be adapted once the new minor release is there.

Reply:
done, thanks